upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -621,6 +621,26 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		}
 	}
 
+	if b.cluster.Spec.Networking.AmazonVPC != nil {
+		key := "networking.amazon-vpc"
+		version := "v0.0.1"
```

Comment:
v0.1.1??  Match the container version

Reply:
I'd also prefer we did `0.1.1-kops.1` - the idea is that the `kops.1` suffix defines the version of our packaging, so if we change e.g. the CPU request, we can do `0.1.1-kops.2`, even if upstream hasn't released `0.1.2`  (the problem being there's nothing after `0.1.1` that isn't `0.1.2...`

Reply:
renamed